specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -4886,6 +4922,114 @@
         "key"
       ]
     },
+    "KnowledgeStore" :
+    {
+      "properties" : {
+        "storageConnectionString": {
+          "type": "string",
+          "description": "The connection string to the storage account projections will be stored in."
+        },
+        "projections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KowledgeStoreProjection",
```

Comment:
There are a couple of places where you have typos, please fix them! (The model validation also lists it)